pyfar/classes/coordinates.py

```diff
@@ -1590,14 +1590,18 @@ def find_within(
             ``'spherical_meter'``
                 distance is determined by the great-circle distance
                 expressed in meters.
-        atol : float
-            Absolute tolerance for distance. The default is 1e-15.
-        radius_tol : float
-            Maximum allowed difference within radii for spherical distance
-            measures. The default is 1e-15.
+        atol : float, None
+            Absolute tolerance for distance. The default is 2*eps, where eps
+            is the resolution of the data type of the coordinates.
         return_sorted : bool, optional
             Sorts returned indices if True and does not sort them if False.
             The default is True.
+        radius_tol : float, None
+            For all spherical distance measures, the coordinates must be on
+            a sphere, so the radius must be constant. This parameter defines
+            the maximum allowed difference within the radii. Note that too
+            large tolerances can lead to wrong results. The default is 2*eps,
```

Comment:
Maybe be more explicit and write:

Note that increasing the tolearance decreases the accuracy of the search, i.e., points that are within the search distance might not be found or points outside the search distance may be returned.